src/lib.rs

```diff
@@ -67,6 +67,8 @@ extern crate uuid;
 extern crate bitflags;
 
 extern crate hyper;
+
+#[cfg(feature = "native-tls")]
```

Comment:
These cfg attributes should be changed to the following:
```rust
#[cfg(feature = "hyper-native-tls")]
```
I'm surprised that this even compiles!

Reply:
switched to hyper-native-tls, thanks